### PR TITLE
Refactor actions and add style to VideoCards

### PR DESCRIPTION
**Pull request by lfaucon**
This PR improve the style of the video card:
- Adds border and shadow frmo Figma's mock-up
- Changes color of Video card actions
- Changes layout of actions to be a row on small screens and a column on large screens
- Add tooltip for Video card actions

Additionally I moved the code of the actions we have, previously they were defined inside other components which makes them impossible to re-use.

Before:
Screenshot 2021-11-06 at 00 31 29

After:
Screenshot 2021-11-06 at 00 31 09

### CODE REVIEW

File: frontend/src/features/videos/VideoList.tsx [hunk 1]
```diff
@@ -26,37 +23,17 @@ function VideoList({
...
     <div>
       {videos.results?.length ? (
         videos.results.map((video: VideoSerializerWithCriteria) => (
           <Grid container className={classes.card} key={video.video_id}>
             <Grid item xs={12}>
-              <VideoCard
-                video={video}
-                actions={[CompareNowAction, AddToRateLaterList]}
-              />
+              <VideoCard video={video} actions={actions} />
```
Comment (sandre35) [inline frontend/src/features/videos/VideoList.tsx:36]: I don't fully understand this change. If the user is not logged in, he can not add videos to his rate later list, neither compare it

**Review by sandre35 — CHANGES_REQUESTED** [review]

File: frontend/src/features/videos/VideoList.tsx (hunk 1 above)
Comment (lfaucon) [reply frontend/src/features/videos/VideoList.tsx:36]: I did not change from the previous behaviour. But you are right. I'll remove the link to comparison

**Review by sandre35 — APPROVED** [review]
Great job! It works very well !!

File: frontend/src/features/videos/VideoList.tsx (hunk 1 above)
Comment (sandre35) [reply frontend/src/features/videos/VideoList.tsx:34]: Indeed it was my mistake at the beginning

**Pull request merged**